Applications/MaterialManager/Contracts/Material/Base/ImageType.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace HomagConnect.MaterialManager.Contracts.Material.Base
+{
+    /// <summary>
+    /// Image types
+    /// </summary>
+    public enum ImageType
```

Comment:
@HOMAG-Boris-Wehrle should we extend and maybe align with types from pM and iD 
![image](https://github.com/HomagGroup/HOMAG-Connect/assets/39241151/0de7e80e-5dda-4203-8305-0060ae8a946b)

Reply:
If necessary please resolve it in a different PR where the models that should be globally available will be moved to the Base Contracts and extended by the necessary values